test/functionalTest/cases/2193_v1_reply_to_v2_response_on_error_with_cpr/forward_using_GET_v2_entities.test

```diff
@@ -0,0 +1,220 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
What about to move all *_in_ngsiv2.test cases in `cases/0787_cpr_full_functional` to `cases/0787_cpr_full_functional_v2` (new directory), then change "_in_ngsiv2" sufix by "ngsiv2_" prefix in all them?

Regarding the test in this PR, I agree in that being new test (not conversions), cases/2193 is the right place to be.